tools/train.py

```diff
@@ -149,7 +149,6 @@ def main():
             PALETTE=datasets[0].PALETTE)
     # add an attribute for visualization convenience
     model.CLASSES = datasets[0].CLASSES
-    meta.update(cfg.checkpoint_config.meta)
```

Comment:
This seems to be necessary, without updating meta, the best ckpt can not be used to test.

Reply:
Why we need to update meta?

Reply:
Right now, meta missing some keys such as `CLASSES` and `PALETTE`, so the best ckpt it saves won't have these keys, which will cause failure during test time. Updating meta will solve the problem.

Reply:
I see. Reverted.